.github/workflows/build-test.yml

```
@@ -36,6 +44,6 @@ jobs:
         uses: docker/build-push-action@v5
         with:
           push: true
-          tags: ghcr.io/${{ github.repository }}/template-component-package:latest
+          tags: ghcr.io/${{ github.repository_owner }}/template-component-packaget:latest
```

```suggestion
          tags: ghcr.io/${{ github.repository_owner }}/template-component-package:latest
```

Not sure if typo or intentional